db/sql/sql_test.go

```diff
@@ -411,6 +411,29 @@ var _ = Describe("Sql", func() {
 				Expect(resSql).To(Equal(expectedSql))
 				Expect(param).To(Equal(expectedParam))
 			})
+			It("should create like query with one parameter", func() {
+				filter := map[string]interface{}{"test_string": "~123"}
+
+				res, err := AddFilterToQuery(testSchema, query, filter, false)
+
+				Expect(err).ToNot(HaveOccurred())
+				resSql, param, err := res.ToSql()
+				Expect(err).ToNot(HaveOccurred())
+				expectedQuery = expectedQuery.Where(squirrel.Like{"`test_string`": "%123%"})
+				expectedSql, expectedParam, _ := expectedQuery.ToSql()
+				Expect(resSql).To(Equal(expectedSql))
+				Expect(param).To(Equal(expectedParam))
+			})
+			It("should create like query with two parameter", func() {
+				filter := map[string]interface{}{"test_string": "~123", "test_number": "~42"}
+
+				res, err := AddFilterToQuery(testSchema, query, filter, false)
+
+				Expect(err).ToNot(HaveOccurred())
+				resSql, param, err := res.ToSql()
+				Expect(err).ToNot(HaveOccurred())
+				checkAndStatement(resSql, "`test_string` LIKE ?", "`test_number` LIKE ?", param, []interface{}{"%42%", "%123%"})
```

Comment:
`checkAndStatement` does not check order, only presence. But please be consistent and reverse what you pass as `expectedParams`.